picas/actors.py

```diff
@@ -9,14 +9,16 @@
 import signal
 import subprocess
 
-from .util import Timer
+from .util import Timer, time_elapsed
 from .iterators import TaskViewIterator, EndlessViewIterator
 
 from couchdb.http import ResourceConflict
-from stopit import threading_timeoutable as timeoutable
+from stopit import ThreadingTimeout
 
 logging.basicConfig(level=logging.INFO)
 log = logging.getLogger(__name__)
+stopit_logger = logging.getLogger('stopit')
```

Comment:
Will this automatically work with the ThreadingTimeout?

Reply:
This is just to avoid logging warnings from stopit.